test/integration/grid-pro-custom-editor.test.js

```diff
@@ -99,86 +99,99 @@ describe('grid-pro custom editor', () => {
+      });
 
-    it('should not stop editing and update value when closing on outside click', async () => {
-      // Open the overlay
-      await sendKeys({ press: 'ArrowDown' });
-      await untilOverlayRendered(cell._content.querySelector('vaadin-date-picker'));
+      it('should not stop editing when clicking inside the overlay but not on focusable element', async () => {
+        // Open the overlay
+        await sendKeys({ press: 'ArrowDown' });
+        await untilOverlayRendered(cell._content.querySelector('vaadin-date-picker'));
 
-      // Move focus back to the input
-      await sendKeys({ press: 'Shift+Tab' });
+        // Click between toolbar buttons
+        const overlayContent = document.querySelector('vaadin-date-picker-overlay-content');
+        const toolbar = overlayContent.shadowRoot.querySelector('[part="toolbar"]');
+        await sendMouseToElement({ type: 'click', element: toolbar });
+        await nextRender();
 
-      // Change single digit to avoid calendar scroll
-      const input = cell._content.querySelector('input');
-      input.setSelectionRange(3, 4);
+        expect(cell._content.querySelector('vaadin-date-picker')).to.be.ok;
+      });
 
-      await sendKeys({ type: '2' });
+      it('should not stop editing and update value when closing on outside click', async () => {
+        // Open the overlay
+        await sendKeys({ press: 'ArrowDown' });
+        await untilOverlayRendered(cell._content.querySelector('vaadin-date-picker'));
 
-      await sendMouse({ type: 'click', position: [10, 10] });
-      await nextRender();
+        // Move focus back to the input
+        await sendKeys({ press: 'Shift+Tab' });
 
-      const editor = cell._content.querySelector('vaadin-date-picker');
-      expect(editor).to.be.ok;
-      expect(editor.value).to.equal('1984-01-12');
+        // Change single digit to avoid calendar scroll
+        const input = cell._content.querySelector('input');
+        input.setSelectionRange(3, 4);
+
+        await sendKeys({ type: '2' });
+
+        await sendMouse({ type: 'click', position: [10, 10] });
+        await nextRender();
+
+        const editor = cell._content.querySelector('vaadin-date-picker');
+        expect(editor).to.be.ok;
+        expect(editor.value).to.equal('1984-01-12');
+      });
     });
 
     describe('mobile', () => {
       let width, height;
 
-      before(() => {
+      before(async () => {
         width = window.innerWidth;
         height = window.innerHeight;
+        await setViewport({ width: 420, height: 600 });
       });
 
       beforeEach(async () => {
-        await setViewport({ width: 420, height: 1020 });
-        // Do not set auto-open-disabled on the date-picker
-        grid.querySelector('vaadin-grid-pro-edit-column').autoOpen = true;
+        grid = createGrid('date', true);
+        await nextRender();
         cell = getContainerCell(grid.$.items, 0, 0);
-        await sendMouse({ type: 'click', position: [10, 10] });
```

Comment:
This was previously needed to close the editor opened in `beforeEach` of the wrapping suite.
Now when we have a separate `desktop` suite, this isn't needed anymore.